src/libraries/Common/src/Internal/Win32Error.cs

```diff
@@ -0,0 +1,119 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+internal static class Win32Error
+{
+    internal static string GetMessage(int errorCode) => errorCode switch
+    {
+        /* ERROR_SUCCESS */                 0 => "The operation completed successfully.",
```

Comment:
The set of error codes that APIs on Windows can return is much larger. Handling of Win32 PAL error codes would work for Win32 PAL on non-Windows, but it is not going to work for Windows.

Reply:
Ah, yes. I initially scoped it to Unix, then relaxed it to test for both, Windows and Unix (mainly was trying to avoid Interop naming, since there is no unsafe/interop aspect in new implementation).